utils/formatters/number.ts

```diff
@@ -151,6 +152,9 @@ export const formatCurrency = (
 		? formatFiatCurrency(value, options)
 		: formatCryptoCurrency(value, options);
 
+export const formatDollars = (value: WeiSource, options?: FormatCurrencyOptions) =>
+	formatCurrency(Synths.sUSD?.name as string, value, { sign: '$', ...options });
```

Comment:
I think we can hardcode 'sUSD' in this one place instead of casting to string because of the optional.

Reply:
Okay, got it, and thanks for your thorough explanation :)